.github/workflows/publish-draft-release.yml

```diff
@@ -0,0 +1,69 @@
+name: Release - Publish draft
+
+on:
+  push:
+    tags:
+      # Catches v1.2.3 and v1.2.3-rc1
+      - v[0-9]+.[0-9]+.[0-9]+*
```

Comment:
TBD: How do we call our tags?

Reply:
Hmm, our branches are `sdk-vx.x.x-polkadot-vxxxx` Let me think...

Reply:
how about the tag is merely: `sdk-vx.y.z`?

Reply:
Then I'd suggest leaving it as is, due to the release title being created with:
- `SDK` (hardcoded)
- tag name as versioning number

Don't think the sdk prepending adds any information to the tag

Reply:
And parity also uses simple version of `vx.y.z`

Reply:
I thought it might be confused with substrate/polkadot if we just use the plain number. but as we are `v0.10.0` already this should not be a problem. So I am ok with the `vx.y.z` format